src/drive/targets/mobile/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "io.cozy.drive.mobile",
+  "version": "1.38.0",
```

Comment:
One more file to edit when upgrading. We have a yarn command: `version:drive`. i think we should update it to support this new file, no?

Reply:
oui, ou supprimer toutes les commandes yarn associées, perso je ne m'en suis jamais servi sur Drive, et ce n'est pas listé dans la doc... que préfères-tu ?

Reply:
les outils pour que ça marche, faut savoir qu'ils sont là, et faut les maintenir. Là on parle de quelque chose qui remplace un SHIFT+CTRL+F ...

bon bref ok je fais la modif...